samples/zigbee/light_switch/src/nus_cmd.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <zephyr.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/services/nus.h>
+#include <logging/log.h>
+#include <settings/settings.h>
+
+#include "nus_cmd.h"
+
+LOG_MODULE_REGISTER(nus_cmd);
```

Comment:
You should use a log level you specified in Kconfig for this module.

Reply:
fixed